### ISSUE TITLE
Add Syntax Highlighting for Operators on Indented Lines

### ISSUE THREAD

**Issue by Git-Christopher** [tidalcycles/atom-tidalcycles#109]
In the current version of the TidalCycles plug-in for Atom, highlighting is not recognized for operators `$` and`#` when indenting a single instance of unbroken code over multiple lines.

For example, this works:

```
d1 $ stack [
    s "cp" #
    speed 0.75
    ]
```
       
while this does not:

```
d1 $ stack [
    s "cp"
    # speed 0.75
    ]
```

Goal of this issue is to have plug-in recognize syntax highlighting in the latter example so to not create antipattern habits in users.

**Comment by ndr-brt** [comment]
should be something into `grammar/tidal.cson` file.
I'm not too good at grammars/regex, but I will take a look :)

**Comment by ndr-brt** [comment]
fixed in v3.10.1
